client/src/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import LoggedOutHome from './components/LoggedOutHome.jsx';
+import Home from './components/Home.jsx';
+// import <Component Name> from './components/<component>.jsx';
+// import <Component Name> from './components/<component>.jsx';
+
```

Comment:
@acenine - Looks awesome! 

The only issue I see is that it looks like this commit is still adding the large  client/dist/bundle.js file. I think you should remove that file from the commit so we don't have to rebase from it. Even if you added it back to the .gitignore, I think you have to remove the file from the commit prior to the merge!